package.json

```diff
@@ -417,6 +416,7 @@
     "identity-obj-proxy": "3.0.0",
     "jest": "26.6.3",
     "jest-environment-jsdom": "^26",
+    "jsonwebtoken": "9.0.0",
```

Comment:
@sneridagh A major semver update for jsonwebtoken and moving it to devDependencies are not backwards compatible, if someone was using it downstream without putting it in package.json explicitly. I would leave this one out of the backport.